portals/publisher/source/src/app/components/Apis/Details/Documents/Listing.jsx

```diff
@@ -516,6 +517,7 @@ class Listing extends React.Component {
                                 <div className={classes.actions}>
                                     <Button
                                         id='add-new-document-btn'
+                                        data-testid='add-document-btn'
```

Comment:
both buttons are for adding a comment. One appear when there are no comments and another appear when there are some comments added. Since, with both situations the automation should be able to add a comment, added the same data-test-id for both 'add comment' buttons